code/index.html

```diff
@@ -1,12 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Project 4: Pizzabot</title>
-  </head>
-  <body>
-    <h1>This is your Pizza Bot</h1>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>Project 4: Pizzabot</title>
+</head>
+
+<body>
+  <h1>This is your Pizza Bot</h1>
+  <button onclick="clicked()">Order Pizza</button>
```

Comment:
A tip is to name the functions with a more describing name! So you can tell what will actually happen when the function is running. Maybe `startOrder()` is better?